tests/ert3/console/integration/test_cli.py

```diff
@@ -63,6 +63,44 @@ def test_cli_init_subfolder(workspace):
             ert3.console._console._main()
 
 
+@pytest.mark.requires_ert_storage
+def test_cli_init_invalid_example(tmpdir, ert_storage):
+    with tmpdir.as_cwd():
+        args = ["ert3", "init", "--example", "something"]
+        with patch.object(sys, "argv", args):
+            with pytest.raises(
+                SystemExit, match="Example something is not a valid ert3 example."
+            ):
+                ert3.console.main()
+
+
+@pytest.mark.requires_ert_storage
+def test_cli_init_example(tmpdir, ert_storage):
+    with tmpdir.as_cwd():
+        args = ["ert3", "init", "--example", "polynomial"]
+        with patch.object(sys, "argv", args):
+            ert3.console.main()
+        polynomial_path = tmpdir / "polynomial"
+        if polynomial_path.exists() and polynomial_path.isdir():
+            # check that copied polynomial example is not an empty directory
```

Comment:
You can probably assert that `stages.yml`, `parameters.yml` and `experiments` also is inside the folder. If not, it does not look like a workspace :)